koku/hcs/database/report_db_accessor.py

```diff
@@ -69,7 +69,7 @@ def get_hcs_daily_summary(self, date, provider, provider_uuid, sql_summary_file,
 
             sql_params = {
                 "provider_uuid": provider_uuid,
-                "year": date.year,
+                "year": date.strftime("%Y"),
                 "month": date.strftime("%m"),
```

Comment:
```suggestion
                "year": date.year,
                "month": date.month,
```
Why don't we do this?

Reply:
Ahhh because `.month` and `.year` don't return strings. Got it.

Reply:
The partitions here are varchars and not integers, using `date.year` or `date.month` get integers back so the comparison fails.
```
>>> type(datetime.today().month)
<class 'int'>
```
This is how the comparison fails: 
```
trino:org1234567> select * from aws_line_items where month = 5;
Query 20230712_192853_03620_44kzd failed: line 1:42: Cannot apply operator: varchar = integer
```